charts/asset-manager/templates/freshclam-cronjob.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ .Release.Name }}-freshclam
+spec:
+  schedule: "0 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          securityContext:
+            seccompProfile:
+              type: RuntimeDefault
+            # Asset Manager needs to run as 2899 since it shares the same NFS volume with
+            # EC2 counterpart
+            runAsUser: 2899
+            runAsGroup: 2899
+            fsGroup: 2899
+          containers:
+            - name: freshclam
+              image: "{{ required "A valid .Values.appImage.repository entry required!" .Values.appImage.repository }}:{{ .Values.appImage.tag }}"
+              imagePullPolicy: {{ .Values.appImage.pullPolicy | default "Always" }}
+              command: ["/bin/sh"]
```

Comment:
Does it really need a shell? Ditto below in worker-deployment.yaml.

Reply:
copy and paste from previous art but I can test that. I believe not.